src/main/resources/templates/user/editUser.html

```diff
@@ -12,6 +12,14 @@
             th:src="@{/js/bootstrap-datetimepicker.min.js}"></script>
     <script type="text/javascript" src="../../js/multivalue.js" th:src="@{/js/multivalue.js}"></script>
     <script type="text/javascript" src="../../js/extension.js" th:src="@{/js/extension.js}"></script>
+    <script type="text/javascript">
+        $(document).ready(function () {
+            $("#externalId_copyFromId").click(function (e) {
```

Comment:
There aren't any HTML ID naming conventions, but I think you shouldn't mix camel case and underscores. I prefer to use hyphens.